components/writers/src/main/java/org/datacleaner/extension/output/CreateExcelSpreadsheetAnalyzer.java

```diff
@@ -71,11 +74,14 @@ public void validate() {
         if (sheetName.indexOf(".") != -1) {
             throw new IllegalStateException("Sheet name cannot contain dots (.)");
         }
+
+        if (file.exists() && !overwriteFile) {
```

Comment:
Well I guess this should not be done here either. But beyond that this is also a bad place to do it - the getSuggestedLabel() method does not have anything to do with validating the component and if it throws an exception here it may have totally different consequences than what you are trying to archieve!